test/experimental/new_embed_test.html

```diff
@@ -39,29 +39,26 @@
 </head>
 
 <body class="d-flex flex-column">
-
-<nav id="navbar" class="UnderlineNav p-2">
-    <div class="UnderlineNav-body">
-        <div class="BtnGroup">
-            <button id="editor-tab" class="btn BtnGroup-item selected" selected type="button">Your code</button>
-            <button id="test-tab" class="btn BtnGroup-item" type="button">Test code</button>
-            <button id="console-tab" class="btn BtnGroup-item" type="button">
-                Output
-                <span id="unread-console-counter" class="Counter" hidden></span>
-            </button>
-        </div>
-    </div>
-    <div class="UnderlineNav-actions">
-        <button id="show-hint" class="btn disabled">Hint</button>
-        <button id="format-code" class="btn">Format</button>
+<div id="navbar" class="tabnav dp-tabs">
+    <nav class="tabnav-tabs">
+        <button id="editor-tab" class="tabnav-tab selected" selected>Your code</button>
+        <button id="solution-tab" class="tabnav-tab" hidden>Solution</button>
+        <button id="test-tab" class="tabnav-tab">Test code</button>
+        <button id="console-tab" class="tabnav-tab">
+            Output
+            <span id="unread-console-counter" class="Counter" hidden></span>
+        </button>
+    </nav>
+    <div class="nav-buttons">
+        <button id="show-hint" class="btn mr-1">Hint</button>
+        <button id="format-code" class="btn mr-2">Format</button>
```

Comment:
this is updating the HTML to match `embed-new.html` (This should have been done as part of the tab UI change)

Reply:
Our coverage is woefully inadequate (I am as much to blame as anyone), which is something that will need to change in a post-I/O world.

Reply:
sure, just something I forgot to do :)

Reply:
sure, just something I forgot to do :)